presentation/src/main/java/com/anwera64/pagodividido/utils/DateFormatter.kt

```diff
@@ -5,7 +5,7 @@ import java.util.*
 
 object DateFormatter {
 
-    private const val DEFAULT_FORMAT = "dd/MM/yyyy"
+    private const val DEFAULT_FORMAT = "MMM dd"
```

Comment:
let's rethink this using year for a future release

Reply:
Sure, we can it back then